composables/useUserAddresses.ts

```diff
@@ -6,21 +6,21 @@ import {
   updateUserAddressMutation,
   deleteUserAddressMutation,
 } from "@/lib/gql/mutations"
-import type { Maybe, CustomerContact } from "@/server/types/GraphQL"
 import type {
   AddUserAddressParams,
   UpdateUserAddressParams,
   DeleteUserAddressParams,
 } from "@/server/types/Api"
+import type { Maybe, CustomerContactCollection } from "@/server/types/GraphQL"
 
 export const useUserAddresses = () => {
   const nuxt = useNuxtApp()
   const fetcher = nuxt.nuxt2Context.$gqlFetch
-  const userShippingAddresses = useState<Maybe<Array<CustomerContact>>>(
+  const userShippingAddresses = useState<Maybe<Array<CustomerContactCollection>>>(
```

Comment:
this type should stay CustomerContact, may need to rebase branch

Reply:
Resolved.